content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -1,5 +1,278 @@
 ---
-title: "Pods"
+reviewers:
+- erictune
+title: Pods
+content_type: concept
 weight: 10
+no_list: true
+card:
+  name: concepts
+  weight: 60
 ---
 
+<!-- overview -->
+
+_Pods_ are the smallest deployable units of computing that can be created and
+managed in Kubernetes.
+
+A _Pod_ (as in a pod of whales or pea pod) is a group of one or more
+{{< glossary_tooltip text="containers" term_id="container" >}} (such as
+Docker containers), with shared storage/network, and a specification
+for how to run the containers.  A Pod's contents are always co-located and
+co-scheduled, and run in a shared context.  A Pod models an
+application-specific "logical host"&mdash;it contains one or more application
+containers which are relatively tightly coupled&mdash;in a pre-container
+world, being executed on the same physical or virtual machine would mean being
+executed on the same logical host.
```

Comment:
```suggestion
A _Pod_ (as in a pod of whales or pea pod) is a group of one or more
{{< glossary_tooltip text="containers" term_id="container" >}}, with shared storage/network resources, and a specification
for how to run the containers. A Pod's contents are always co-located and
co-scheduled, and run in a shared context. A Pod models an
application-specific "logical host": it contains one or more application
containers which are relatively tightly coupled. 
In non-cloud contexts, applications executed on the same physical or virtual machine are analogous to cloud applications executed on the same logical host.
```

I think I got the gist of what you were trying to say here, but I might be off base. Feel free to ignore all/parts of this suggestion.

Reply:
These aren't actually my words. Happy to tweak them though.